Dockerfile

```diff
@@ -11,6 +11,7 @@ MAINTAINER ICGC <dcc-support@icgc.org>
 ENV EGA_VERSION 2.2.2
 ENV GT_VERSION 3.8.7
 ENV GT_VERSION_LONG 207
+ENV GDC_VERSION gdc-client_v1.0.1_Ubuntu14.04_x64
```

Comment:
I was being a little cautious because I recently had an error where the url was changed from `gdc-client_v1.0.1_Ubuntu14.04_x64_0` to `gdc-client_v1.0.1_Ubuntu14.04_x64`.